app/src/main/java/org/mozilla/focus/activity/MainActivity.kt

```diff
@@ -80,6 +86,8 @@ open class MainActivity : LocaleAwareAppCompatActivity() {
         }
         setContentView(R.layout.activity_main)
 
+        setSplashScreenPreDrawListener()
```

Comment:
After some manual testing, I was experiencing an empty screen for a couple of seconds before, I seeing the onboarding UI. 
https://user-images.githubusercontent.com/773158/160718264-c810796f-66e3-4467-ae2e-de719d3d0418.mp4

Reply:
It seems that Google already knows about this, but marked it as expected behaviour https://issuetracker.google.com/issues/205021357

Reply:
I believe we may need to build our own splash screen, as many users will be affected by this issue. Do they have a fix on an alpha channel of the library?

Reply:
From what I read...they don't. Yeah, we can add our own splash screen and I will get documented about how can we disable the default one which is added starting with Android 12

Reply:
I think it will be the best call, we could move that work in separate PR to unblock the other changes from landing.

Reply:
From what I found until now we can't disable the default splash screen for android 12. To move that work in a separate pr means to also remove `setSplashScreenPreDrawListener()` from this pr, which will affect the entire pr.
In my opinion we should also land this commit and search after that for a solution, because it seems to happens when the app is started from other applications or android studio